web/src/components/__tests__/HeaderLogo.spec.tsx

```diff
@@ -20,8 +20,8 @@ describe('HeaderLogo', () => {
   })
 
   it('should show the regular header app icon if there is no campaign', () => {
-    jest.setSystemTime(1615374110000) // Wed Mar 10 2021 11:01:50 GMT+0000
-    config.campaign = undefined
+    jest.setSystemTime(11686641878) // Tue Jun 13 2023 07:37:58 GMT+0000
+    config.campaign = womensDayCampaign
```

Comment:
Why did you change this? Please revert

Reply:
@steffenkleinle it cannot read properties of undefined (reading 'startDate')

Reply:
If you apply my proposed changes in `HeaderLogo.tsx` you can revert this, just tested it.